src/compiler.ts

```diff
@@ -1,11 +1,18 @@
 import { workspace } from 'vscode';
 import * as prql from 'prql-js';
+import * as constants from './constants';
 
 export function compile(prqlString: string): string | ErrorMessage[] {
   // create compile options from prql workspace settings
   const compileOptions = new prql.CompileOptions();
   const target = <string>workspace.getConfiguration('prql').get('target');
+  const addCompilerInfo: boolean = <boolean>(
+    workspace
+      .getConfiguration('prql')
+      .get(constants.AddCompilerSignatureComment)
```

Comment:
@max-sixty I really don't like your pre-commit hooks reformatting multi-line statements.

Can we disable it and just use eslint pre-check?

I find pre-commit very obtrusive and overwriting md formatting too.